src/ngraph/op/experimental/quantized_conv.hpp

```diff
@@ -33,7 +33,10 @@ namespace ngraph
                                  const CoordinateDiff& padding_below,
                                  const CoordinateDiff& padding_above,
                                  const Strides& data_dilation_strides,
-                                 const std::shared_ptr<Node> scale);
+                                 const std::shared_ptr<Node> scale,
+                                 const std::shared_ptr<Node> input_scale = nullptr,
```

Comment:
Similar issue to my other comment: I don't think we should have a default value of `nullptr` here, because this is just going to segfault once `check_single_output_args` is called.